internal/flags.go

```diff
@@ -184,6 +184,11 @@ func NewApp() (app *cli.App) {
 				Usage: "Reduce S3 operation costs at the expense of some performance (default: off)",
 			},
 
+			cli.BoolFlag{
+				Name:  "no-dir-compat",
```

Comment:
gcsfuse uses --implicit-dirs for the inverse behavior and that's the closest I could find, maybe call this --explicit-dirs or --no-implicit-dirs for consistency?